linalg/ode.hpp

```diff
@@ -107,6 +109,13 @@ class ODESolver
    {
       mfem_error("ODESolver has no state vectors");
    }
+   virtual void  SetStateVector(Vector &state)
+   {
+      mfem_error("ODESolver has no state vectors");
+   }
+
+   static const char *ODETypes;
+   static ODESolver *Select(const int ode_solver_type);
```

Comment:
I recommend adding a comment here noting the user that the ownership/management of the created `ODESolver` object is assumed to be handled by the calling scope.